libs/openchallenges/styles/src/lib/_general.scss

```diff
@@ -1,10 +1,13 @@
 @use 'libs/openchallenges/styles/src/lib/constants';
 
-html,
+html
```

Comment:
does it work without comma between `html` and `body`?

Reply:
That brings up a good point - we don't necessarily need these defs for both `html` and `body`.